Sources/Utility/Process.swift

```diff
@@ -77,16 +78,25 @@ public final class Process: ObjectIdentifierProtocol {
 
     /// The process id of the spawned process, available after the process is launched.
     public private(set) var processID = ProcessID()
-
-    /// The result of the process execution. Available after process is terminated.
-    public private(set) var result: ProcessResult?
-
-    /// If the subprocess has launched.
+    
+    /// If the subprocess has launched, this property is not thread safe.
```

Comment:
This is technically thread safe for clients. I meant writing this as a comment (not a doc-comment), something like this:
```
/// If the subprocess has launched.
// Note: This property is not protected by the serial queue because it is only mutated in `launch()`, which will be called only once.
```